app/assets/src/components/ui/notifications/Notification.jsx

```diff
@@ -20,19 +20,21 @@ class Notification extends React.Component {
   }
 
   render() {
+    const { children, className, displayStyle, onClose, type } = this.props;
     return (
       <div
         className={cx(
-          this.props.className,
+          className,
           cs.notification,
-          cs[this.props.type]
+          cs[type],
+          displayStyle && cs.flat
```

Comment:
It looks like this just always applies `cs.flat`. Is that intentional?

Reply:
Not at all! Fixed